lib/helpers/getBasicDomNodeProfile.js

```diff
@@ -0,0 +1,19 @@
+import { getDomNodeAttributes } from './getDomNodeAttributes';
+
+const EXCLUDE_VALUE_REGEX = /^((?!value).)*$/;
+
+export function getBasicDomNodeProfile(el) {
```

Comment:
Just wanted to say that this way of handling the recursion is much, much clearer than it was.